api/scpca_portal/models/computed_file.py

```diff
@@ -89,6 +89,10 @@ def get_local_sample_metadata_path(sample, download_config: Dict) -> Path:
         )
         return common.OUTPUT_DATA_PATH / "_".join(file_name_parts)
 
+    @staticmethod
+    def get_local_portal_metadata_path() -> Path:
+        return common.OUTPUT_DATA_PATH / common.PORTAL_METADATA_COMPUTED_FILE_NAME
```

Comment:
```suggestion
    @staticmethod
    def get_local_file_path(download_config: Dict) -> Path:
        """ Takes a download_config dictionary and returns the filepath where the zipfile will be saved locally before upload. """
        if download_config is common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG:
            return common.OUTPUT_DATA_PATH / common.PORTAL_METADATA_COMPUTED_FILE_NAME
```

We should consolidate these and standardize how we get the value. For now let's keep how project and sample computed file names/ paths are evaluated on their models. We can move them under here later.